schema/module_schema.go

```diff
@@ -113,6 +115,22 @@ func schemaForDependentModuleBlock(localName string, modMeta *module.Meta) (*sch
 		}
 	}
 
+	moduleSourceRegistry, err := tfaddr.ParseRawModuleSourceRegistry(module.SourceAddr)
+	if err == nil {
+		version := module.Version
+		if version == "" {
+			version = "latest"
+		}
+		bodySchema.DocsLink = &schema.DocsLink{
+			URL: fmt.Sprintf(
+				`https://%s/modules/%s/%s`,
+				moduleSourceRegistry.PackageAddr.Host.ForDisplay(),
```

Comment:
Are you sure that private/other Registries use the same URL structure? If not, it may be safer to just allow `registry.terraform.io` here and avoid linking to any other ones.

Secondly if we end up using the `Host` here would it be safer to just use the raw hostname via `String()`? I'm guessing that `ForDisplay()` is really meant _for display_ - i.e. human readable version of the hostname to show in error messages etc. which doesn't look like something we should be using in URLs.

Reply:
That's a valid point. I was hoping we could support any external registry with this change. But other registries might not even offer a web UI for docs at all. So I'm limiting it to `registry.terraform.io` for now.